ee/ucp/deploy-application-package.md

```diff
@@ -17,14 +17,14 @@ Once an application package has been deployed, you manipulate and manage it as y
 
 ## Creating a stack in the UCP web interface
 
-To create a stack in the UCP web interface, follow these steps:
+To create a stack in the UCP web interface:
 
 1. Go to the UCP web interface.
-2. In the lefthand menu, first select **Shared Resources**, then **Stacks**.
+2. From the left menu, select **Shared Resources** > **Stacks**.
 
-    ![Create stacks in UCP](/ee/ucp/images/ucp-create-stack.png)
+    ![Create stacks in UCP](/ee/ucp/images/v32stacks.png)
 
-3. Select **Create Stack** to display **1. Configure Application** in the stack creation dialog.
+3. Select **Create Stack** to open the **Create Application** window.
```

Comment:
Change wording to read, "Click the **Create Stack** button located at the right of the screen to open the **Create Application** window."